tests/phpunit/tests/post.php

```diff
@@ -1637,4 +1637,53 @@ function test_wp_resolve_post_date() {
 		$resolved_post_date = wp_resolve_post_date( $invalid_date, $invalid_date );
 		$this->assertFalse( $resolved_post_date );
 	}
+
+	/**
```

Comment:
In these tests it would be good to include some type casting:

```php
// Option values: [ 1, '1', 2 ]
stick_post( '1' );
unstick_post( '1' );
```
Maybe some floats too if you are so inclined.

It would also be good to limit each test to a single assertion (or as few as possible). A test stops running if a single assertion fails, keeping the number low helps identify when multiple errors are occurring.